uefi/firmwarevolume.go

```diff
@@ -37,16 +37,16 @@ type Block struct {
 // FirmwareVolumeFixedHeader contains the fixed fields of a firmware volume
 // header
 type FirmwareVolumeFixedHeader struct {
-	Zeros          [16]uint8
+	Zeros          [16]uint8 `json:"-"`
 	FileSystemGUID [16]uint8
 	Length         uint64
 	Signature      uint32
 	AttrMask       uint8
 	HeaderLen      uint16
 	Checksum       uint16
-	Reserved       [3]uint8
+	Reserved       [3]uint8  `json:"-"`
```

Comment:
and, fwiw, intel often calls things unused, and reserved, interchangeably, then uses them anyway. See: 386 PTEs.

Reply:
If I remember correctly, I do see some things in this field when I parse an actual rom image, but we ignore them right now